console/tsconfig.json

```diff
@@ -8,7 +8,8 @@
     "strict": true,
     "esModuleInterop": true,
     "noEmitOnError": false,
-    "noEmit": true
+    "noEmit": true,
+    "types": ["react"]
```

Comment:
This issues's workaround.
https://github.com/DefinitelyTyped/DefinitelyTyped/issues/33311